database/database.js

```diff
@@ -1,8 +1,18 @@
 const { Pool } = require("pg");
-const pool = new Pool({
-    user: process.env.DB_USERNAME,
-    password: process.env.DB_PASSWORD,
-    database: process.env.DATABASE
-})
+
+const config = process.env.DATABASE_URL
```

Comment:
condition to using DATABASE_URL if it is defined. Heroku PostgreSQL define a env var with the database credentials. if we don't have it, we're following the regular way (development mode)